README.md

```diff
@@ -1,119 +1,23 @@
-+ Names which are in scope in a type are also always in scope in the body of
-  the corresponding definition.
-+ Better inference. Holes are global to a source file, rather than local to
-  a definition, meaning that some holes can be left in function types to be
-  inferred by the type checker. This also gives better inference for the types
-  of `case` expressions, and means fewer annotations are needed in interface
-  declarations.
-+ Better type checker implementation which minimises the need for compile
-  time evaluation.
-+ New Chez Scheme based backend which both compiles and runs faster than the
-  default Idris 1 backend. (Also, optionally, Racket and Gambit can be used
-  as targets).
-+ Everything works faster :).
-
-A significant change in the implementation is that there is an intermediate
-language `TTImp`, which is essentially a desugared Idris, and is cleanly
-separated from the high level language which means it is potentially usable
-as a core language for other high level syntaxes.
-
-JavaScript
-----------
-The JavaScript codegen uses the new BigInt, hence Node.js 10.4 or higher is required.
-
-Editor Plugins
---------------
-The [wiki lists the current plugins available for common text editors](
-https://github.com/idris-lang/Idris2/wiki/1-%5BLanguage%5D-Editor-support)
-and their features.
-
-Things still missing
---------------------
-
-+ Cumulativity (so we currently have Type : Type! Bear that in mind when you
-  think you've proved something :))
-+ 'rewrite' doesn't yet work on dependent types
-
-Contributions wanted
--------------------
-
-+ [Contribution guidelines](CONTRIBUTING.md)
-+ [Good first issues](https://github.com/idris-lang/Idris2/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22)
-+ [Contributors wanted](https://github.com/idris-lang/Idris2/wiki/2-%5BCommunity%5D-Contributions-wanted)
-
-If you want to learn about Idris more, contributing to the compiler could be one
-way to do so. Just select one good first issue and ask about it on the [Discord](https://discord.gg/UX68fDs2jc) channel.
-
-Talks
------
-
-[![Edwin Brady Tells Us What's New in Idris 2 (Berlin Functional Programming Group)](https://img.youtube.com/vi/nbClauMCeds/0.jpg)](https://www.youtube.com/watch?v=nbClauMCeds "Edwin Brady Tells Us What's New in Idris 2 (Berlin Functional Programming Group)")
-
-[![Scheme Workshop Keynote (ACM SIGPLAN)](https://img.youtube.com/vi/h9YAOaBWuIk/0.jpg)](https://www.youtube.com/watch?v=h9YAOaBWuIk "Scheme Workshop Keynote (ACM SIGPLAN)")
-
-[![Idris 2 - Type-driven Development of Idris (Curry On - London 2019)](https://img.youtube.com/vi/DRq2NgeFcO0/0.jpg)](https://www.youtube.com/watch?v=DRq2NgeFcO0 "Idris 2 - Type-driven Development of Idris (Curry On - London 2019)")
-
-[![Idris 2: Type-driven Development of Idris (Code Mesh LDN 18)](https://img.youtube.com/vi/mOtKD7ml0NU/0.jpg)](https://www.youtube.com/watch?v=mOtKD7ml0NU "Idris 2: Type-driven Development of Idris (Code Mesh LDN 18)")
```

Comment:
Please keep these in the file, they're useful introductions and showcases for what Idris2 is and can do.

We should probably add the SPLV'20 playlist to the mix (https://www.youtube.com/watch?v=2pa3oRFNO8E&list=PLmYPUe8PWHKqBRJfwBr4qga7WIs7r60Ql)

Reply:
OK. I do think they should appear differently. Currently the thumbnails don't give any information about the content and fill a lot of space

Regarding useful introductions, should we also mention the official tutorials etc for those who prefer to read than listen?

Reply:
here's how I presented them in the Resources wiki. It's another possibility

https://github.com/idris-lang/Idris2/wiki/2-%5BCommunity%5D-Resources#learning